src/MergeContext.js

```diff
@@ -1795,13 +1794,8 @@ class PullRequest {
 }
 
 // promises to update/advance the given PR, hiding PullRequest from callers
-function Process(rawPr, banStaging) {
+export default function Process(rawPr, banStaging) {
```

Comment:
If possible without a lot of changes, let's force callers to specify explicitly what they want to import:

```suggestion
export function Process(rawPr, banStaging) {
```

Reply:
Ok.